docs/process/release.md

```diff
@@ -74,6 +74,8 @@ Releasing is performed using Angular's unified release tooling. Each week, two r
 After FW releases `-rc.0` for an upcoming minor/major version, update the corresponding version in:
 
 - [`latest-versions.ts`](/packages/schematics/angular/utility/latest-versions.ts#L=18)
+- [`latest-versions/package.json`](/packages/schematics/angular/utility/latest-versions/package.json)
+  (only needed for a major release)
```

Comment:
This may be needed for minors too.

Reply:
I thought the `^` in the semver meant it would automatically work for minors? Am I misunderstanding how these work?

Reply:
Ah I think I see, this will always be updated to something like `^15.1.0-next` first, and then switched to `^15.1.0` right before the release, so this still applies to minors. Removed the "major release" comment.